package.json

```diff
@@ -13,6 +13,9 @@
     "url": "https://github.com/formidablelabs/victory.git"
   },
   "private": true,
+  "engines": {
```

Comment:
Note: Since this is a private package.json that isn't published, should be safe for us. (We wouldn't want to restrict yarn engines on like our published packages...)